### PR TITLE
Jfrog platform 1.3.0

### PR DESCRIPTION
**Pull request by MahithaB**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by estherk15 — COMMENTED** [review]
Hi @MahithaB thanks for your work on this! I left some recommendations and suggestions. Let me know if you have any questions on this. There's room to format this documentation with tabs, similar to [Aerospike](http://localhost:1313/integrations/aerospike/?tab=host#configuration), I can help with that. Can you confirm this as the information outline:
FluentD Installation
  a. OS/Virtual Machine
      i. Gem based install
      ii. Configure FluentD
  b. Docker
  c. Kubernetes
  d. Artifactory, Artifactory-HA and Xray ENV variables (I think we can pull this out instead of repeating it multiple times.)

File: jfrog_platform_self-hosted/README.md
Comment (MahithaB) [reply jfrog_platform_self-hosted/README.md:27]: @estherk15 Gem based install is one of the suggested forms of installation just like CentOS/ MacOS etc. And configure fluentd section is common to any form of installation mentioned in the table above. That is the reason why we had Configure Fluentd as a sub section that is common to all OS/VM installations

**Comment by MahithaB** [comment]
@estherk15
d. Artifactory, Artifactory-HA, and Xray ENV variables (I think we can pull this out instead of repeating it multiple times.)

The reason for not abstracting out the env variable section is that we've had feedback/customer support tickets in the past where users had difficulty setting environment variables because Artifactory-HA section for example doesn't have steps to configure .env vars under it

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [reply jfrog_platform_self-hosted/README.md:28]: @MahithaB What if we move Configure Fluentd to the top and move Gem based install below that so readers can still access both. Having a header for Gem based install, gives users the option to link to it or share links to it.

Comment (MahithaB) [reply jfrog_platform_self-hosted/README.md:28]: @estherk15 Configuring Fluentd step is something that has to be done after the OS/VM or Gem-based installation is done. So, moving it up might add confusion. Also gem based installation is a form of installation that can be done on any VM, so it being a part of OS / VM section makes it easier

Comment (estherk15) [reply jfrog_platform_self-hosted/README.md:28]: That makes sense! To confirm, the information structure is OS/Virtual Machine and both the Gem based install and Configure FluentD are subsections.
```
OS/Virtual Machine
- Gem based install
- Configure FluentD
```

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [inline jfrog_platform_self-hosted/README.md:11]:
```suggestion
* Install the JFrog Platform (Self-hosted) integration.
```

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [inline jfrog_platform_self-hosted/README.md:233]:
```suggestion
#### JFrog Xray Violations dashboard
```

**Review by estherk15 — COMMENTED** [review]
Thanks for the edits @MahithaB! I found a few more edits for consistency. Can you also confirm you've seen the other comments/suggestions in the Files Changed tab, for example [this comment](https://github.com/DataDog/integrations-extras/pull/1895/files?file-filters%5B%5D=.csv&file-filters%5B%5D=.example&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.py&file-filters%5B%5D=.toml&file-filters%5B%5D=.txt&file-filters%5B%5D=.yaml&file-filters%5B%5D=.yml&owned-by%5B%5D=estherk15&show-deleted-files=true&show-viewed-files=true#r1243800140)

File: jfrog_platform_self-hosted/README.md
Comment (MahithaB) [reply jfrog_platform_self-hosted/README.md:79]: They're actually a continuous set of commands. So, makes sense to number them accordingly

**Comment by MahithaB** [comment]
@estherk15 thank you for the review. I was just looking at the checks and I don't see any specific error in Validate Repository /run/validate (pull_request) check but it throws an error. Do I need to fix something on my end?

**Comment by estherk15** [comment]
> @estherk15 thank you for the review. I was just looking at the checks and I don't see any specific error in Validate Repository /run/validate (pull_request) check but it throws an error. Do I need to fix something on my end?



> @estherk15 thank you for the review. I was just looking at the checks and I don't see any specific error in Validate Repository /run/validate (pull_request) check but it throws an error. Do I need to fix something on my end?

Hi @MahithaB , this might be a configuration issue but I'm not as sure of this. After the other reviewers look through your PR, they may have better insight on this check.

File: jfrog_platform_self-hosted/CHANGELOG.md
Comment (nmuesch) [inline jfrog_platform_self-hosted/CHANGELOG.md:5]: nit:
```suggestion
* Modified installation steps for sending logs, metric and violations data to Datadog
```

File: jfrog_platform_self-hosted/manifest.json
Comment (nmuesch) [inline jfrog_platform_self-hosted/manifest.json:78]: In order to provide a smooth experience for users, can you continue to use the same `key` as the old manifest file? 
Ex: `Jfrog Artifactory Dashboard`. 

Unfortunately we cannot remove existing dashboards or change the `key` at this time. However the changes to the dashboard.json payloads themselves are still OK!

**Review by nmuesch — CHANGES_REQUESTED** [review]
Two quick comments from my review. Let me know if you have any questions!

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [inline jfrog_platform_self-hosted/README.md:11]: By the nature of numbered lists, it might not be necessary to include this. If you are seeing this as a common issue for customers:
```suggestion
Follow the instructions in the order listed:
```

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [inline jfrog_platform_self-hosted/README.md:7]: Instead of placing the deprecation note here, recommend using an alert at the top of the page under Overview section above.
```suggestion
<div class="alert alert-warning">The JFrog Platform (self-hosted) check replaces the <a href="https://docs.datadoghq.com/integrations/jfrog_platform/">JFrog Platform check</a> which is deprecated. JFrog metrics are gathered by FluentD.</div>
```

File: jfrog_platform_self-hosted/README.md
Comment (MahithaB) [reply jfrog_platform_self-hosted/README.md:2]: @estherk15 doesn't updating the README update https://docs.datadoghq.com/integrations/jfrog_platform as well? If that is the case, linking it might confuse the customers

Comment (MahithaB) [reply jfrog_platform_self-hosted/README.md:2]: Instead, I can add 

<div class="alert alert-warning">The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will be deprecated by July 31st 2023.</div>

Because we're using Fluentd and the check that is in place right now (JFrog Platform (self-hosted)) check will be removed by the end of this month.

Comment (estherk15) [reply jfrog_platform_self-hosted/README.md:4]: Thanks for the clarification, I may have been confused on what is being deprecated! 
Suggested wording: "Agent check will deprecate on July 31, 2023."

Comment (MahithaB) [reply jfrog_platform_self-hosted/README.md:5]: changed it, can you please review again

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [inline jfrog_platform_self-hosted/README.md:1]:
```suggestion
```

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [inline jfrog_platform_self-hosted/README.md:2]: Formatting for an alert div
```suggestion
<div class="alert alert-warning">The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will deprecate on July 31st 2023.</div>
```

File: jfrog_platform_self-hosted/README.md
Comment (estherk15) [reply jfrog_platform_self-hosted/README.md:5]: Thanks @MahithaB, just made a suggestion for formatting, approved for documentation!

**Review by estherk15 — APPROVED** [review]

**Comment by MahithaB** [comment]
@estherk15 thank you so much for the review, all the suggestions are implemented

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:224]:
```suggestion
* **Audit** - This section tracks audit logs that help you determine who is accessing your Artifactory instance and from where. These can help you track potentially malicious requests or processes (such as CI jobs) using expired credentials.
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:225]:
```suggestion
* **Requests** - This section tracks HTTP response codes and the top 10 IP addresses for uploads and downloads.
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:228]:
```suggestion
This dashboard tracks Artifactory System Metrics, JVM memory, Garbage Collection, Database Connections, and HTTP Connections metrics.
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:231]:
```suggestion
This dashboard provides a summary of access, service and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight.
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:234]:
```suggestion
This dashboard provides an aggregated summary of all the license violations and security vulnerabilities found by Xray. Information is segmented by watch policies and rules. Trending information is provided on the type and severity of violations over time, as well as insights on most frequently occurring CVEs, top impacted artifacts and components.
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:237]:
```suggestion
This dashboard tracks System Metrics and data metrics about Scanned Artifacts and Scanned Components.
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:217]: I would move this section to the Overview section of the readme.

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:252]:
```suggestion
**Q : I have a JFrog Cloud Enterprise + license. Will I be able to use this integration?**
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:254]:
```suggestion
A: No. This integration is only built for JFrog customers who have the on-prem or self-hosted JFrog subscription. JFrog is working on a SaaS Log streaming solution which will allow our SaaS customers to stream logs to Datadog. We hope to launch that solution in late 2023.
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:256]:
```suggestion
**Q : I am about to upgrade from on-prem to JFrog Cloud. Can I expect all the same logs to stream into Datadog from my SaaS instance post-migration when I install the SaaS version of the integration?**
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:258]: This sentence is incomplete

File: jfrog_platform_self-hosted/assets/dashboards/jfrog_artifactory_logs_(self-hosted).json
Comment (anawishnoff) [inline jfrog_platform_self-hosted/assets/dashboards/jfrog_artifactory_logs_(self-hosted).json:1]: It looks like this dashboard is has a screenboard layout. Can you please convert all dashboards to a dashboard layout?

File: jfrog_platform_self-hosted/manifest.json
Comment (anawishnoff) [inline jfrog_platform_self-hosted/manifest.json:23]:
```suggestion
```
The logo does not need to be added here, it will automatically be added to your tile.

File: jfrog_platform_self-hosted/manifest.json
Comment (anawishnoff) [inline jfrog_platform_self-hosted/manifest.json:42]: Please ensure all images fit our [media guidelines](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel) - specifically, all images must have a 16:9 aspect ratio.

File: jfrog_platform_self-hosted/manifest.json
Comment (anawishnoff) [inline jfrog_platform_self-hosted/manifest.json:52]:
```suggestion
      "Supported OS::macOS",
      "Offering::Integration"
```
Please also add submitted data types in the following format:
`"Submitted Data Type::Metrics"`

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Mostly wording updates and general comments.

The setup section is quite long, have you considered perhaps hosting these on a doc on your website so the user can easily navigate to the sections that are relevant to them?

File: jfrog_platform_self-hosted/assets/dashboards/jfrog_xray_logs_(self-hosted).json
Comment (nmuesch) [inline jfrog_platform_self-hosted/assets/dashboards/jfrog_xray_logs_(self-hosted).json:15]: Can you replace these URLs with:

```suggestion
                "url": "https://raw.githubusercontent.com/DataDog/integrations-extras/master/jfrog_platform/assets/images/xray.png",
```

This link won't work until this PR is merged, but any github links should point to the Datadog org copy instead of the fork (it looks like there's a couple cases of this)

File: jfrog_platform_self-hosted/CHANGELOG.md
Comment (nmuesch) [inline jfrog_platform_self-hosted/CHANGELOG.md:2]: Non-blocking nit: This comment was probably added a while back. We actually now show the CHANGELOG directly in the in-app tile! So this sentence can probably be removed since it doesn't make sense in all contexts where this data is shown.

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:42]:
```suggestion
We recommend following the installation guide that matches your environment:
```

File: jfrog_platform_self-hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/README.md:66]:
```suggestion
A: At launch, the SaaS version of the integration will only stream the artifactory-request, access-audit and access-security-audit logs from your SaaS JFrog instance to Datadog.
```

File: jfrog_platform_self-hosted/manifest.json
Comment (anawishnoff) [inline jfrog_platform_self-hosted/manifest.json:49]:
```suggestion
      "Submitted Data Type::Metrics",
      "Submitted Data Type::Logs"
```

**Review by anawishnoff — APPROVED** [review]
Thanks for making all of these updates! This looks good to me. Once all approvals are in and the log pipeline is complete we can merge :)

**Review by alopezz — CHANGES_REQUESTED** [review]
Please rebase against master.

Apart from that:

- Can anyone provide context on why it was necessary to rename `jfrog_platform` to `jfrog_platform_self-hosted`?
- Why have tests been dropped?

**Comment by MahithaB** [comment]
@alopezz 

1. The reason jfrog_platform is renamed to jfrog_platform_self-hosted is because this integration concentrates on self-hosted versions and we're working on a new integration that covers the Saas versions
2. Tests have been dropped because we moved away from using datadog-agent to gather any information.

**Comment by alopezz** [comment]
> 2. Tests have been dropped because we moved away from using datadog-agent to gather any information.

In that case, you should drop the implementation as well, right?

**Comment by MahithaB** [comment]
@alopezz we've set a deprecation notice that the implementation for the existing customers will be in place till the end of August. And we've removed it from the current setup instructions already

File: jfrog_platform_self-hosted/CHANGELOG.md
Comment (anawishnoff) [inline jfrog_platform_self-hosted/CHANGELOG.md:6]: Please update the changelog to use this format:

```suggestion
# CHANGELOG - JFrog Platform Self-Hosted

## 1.3.0 / 2023-08-11

***Fixed***:
* Modified installation steps for sending logs, metrics and violations data to Datadog
* Supporting only OS/VM, docker and k8s forms of installation for JFrog Platform
* Updated documentation

```

Comment (anawishnoff) [reply jfrog_platform_self-hosted/CHANGELOG.md:6]: This should resolve your validation issues!

Comment (MahithaB) [reply jfrog_platform_self-hosted/CHANGELOG.md]: @anawishnoff I've updated the CHANGELOG and the validate repository is still not successful 😢

Comment (nmuesch) [reply jfrog_platform_self-hosted/CHANGELOG.md]: Hey @MahithaB There was indeed an issue with `ddev` that got fixed and released. I've re-run the CI and the latest failure is vaild: its failing because the CODEOWNERS file will need to be updated to reflect the new integration folder name. Once thats updated the CI should pass again. Sorry for the inconvenience there!

Comment (MahithaB) [reply jfrog_platform_self-hosted/CHANGELOG.md]: @nmuesch thank you. I've changed the CODEOWNERS file to the new integration folder name. I think there is a new error now. Can you please give me more information on that?

Comment (nmuesch) [reply jfrog_platform_self-hosted/CHANGELOG.md]: @MahithaB The service_checks.json file is required even if its just an empty json object. Can you put that file back and add this to the manifest:

```
      "service_checks": {
        "metadata_path": "assets/service_checks.json"
      }
```

**Review by ofek — CHANGES_REQUESTED** [review]
Hello! The root directory needs to be renamed from `jfrog_platform_self-hosted` to `jfrog_platform_self_hosted` and the directory under `datadog_checks` needs to be named the same

File: jfrog_platform_self-hosted/CHANGELOG.md
Comment (MahithaB) [reply jfrog_platform_self-hosted/CHANGELOG.md]: @nmuesch Got it, added the service_checks back

**Comment by MahithaB** [comment]
@ofek I've renamed the root directory, and the directory inside datadog_checks to jfrog_platform_self_hosted. Also renamed the CODEOWNERS file to reflect the same. Thanks!

File: jfrog_platform_self-hosted/CHANGELOG.md
Comment (MahithaB) [reply jfrog_platform_self-hosted/CHANGELOG.md]: @nmuesch  Can you give info the next validation issue

Comment (nmuesch) [reply jfrog_platform_self-hosted/CHANGELOG.md]: Looks like the folder rename requires the codecov file to need to be updated. From the error it looks like you can run `ddev validate ci --sync` and commit the changes to get past it

Comment (MahithaB) [reply jfrog_platform_self-hosted/CHANGELOG.md]: @nmuesch can I get more info on the next error please

Comment (nmuesch) [reply jfrog_platform_self-hosted/CHANGELOG.md]: This one looks in Ofeks territory so I may not be able to help with specifics. But I think it's because the configuration spec.yaml file is deleted and not in the new folder.

Comment (MahithaB) [reply jfrog_platform_self-hosted/CHANGELOG.md]: @ofek can I get more information on the current error? Thanks!

**Comment by ofek** [comment]
You will need to reintroduce this file https://github.com/DataDog/integrations-extras/blob/master/jfrog_platform/assets/configuration/spec.yaml

More info: https://datadoghq.dev/integrations-core/meta/config-specs/#schema

**Comment by MahithaB** [comment]
@ofek I've added the spec.yaml file back and the validate repository still fails. Can you please give more information on the error? Thanks!

**Review by ofek — APPROVED** [review]

File: jfrog_platform_self_hosted/README.md
Comment (anawishnoff) [inline jfrog_platform_self_hosted/README.md:50]:
```suggestion
### Metrics
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
After reviewing the tile, I just had a few last minor comments. Once these are addressed and the log pipeline is complete, this should be good to go. Thank you for all of your hard work here!

**Review by anawishnoff — APPROVED** [review]

**Review by emarsha94 — APPROVED** [review]

**Review by anawishnoff — APPROVED** [review]

**Comment by nmuesch** [comment]
Hey @MahithaB Looks like all required reviews are in and everything is looking good! 

There just seems to be some merge conflicts that need to be addressed before we're able to merge. Could you please resolve those?

File: jfrog_platform_self_hosted/README.md
Comment (emarsha94) [inline jfrog_platform_self_hosted/README.md:1]: @MahithaB This line needs to be updated to the new date that Shailendra agreed on.

**Review by emarsha94 — CHANGES_REQUESTED** [review]

**Review by emarsha94 — APPROVED** [review]

**Pull request merged**
